scripts/common/preflights.sh

```diff
@@ -583,6 +583,13 @@ function host_preflights() {
                 ;;  
             2)
                 logWarn "Host preflights have warnings"
+                logWarn "It is highly recommended sort out the warnings conditions before continue."
+                logWarn "Be aware that continue with preflight warnings can result in failures."
+                log ""
+                logWarn "Would you like to continue? "
+                if ! confirmN ; then
```

Comment:
I'd like to call out that this will now cause kURL installations to fail by default when run via scripts or in non-interactive terminals when warnings are encountered. We should run this by @kyleprice before making this change and call it out in the release notes. Otherwise we could take a less drastic approach and change this to `confirmY`. I think we should discuss this with the team either way and determine the desired change.

```release-note
- kURL installations will now fail by default for non-interactive terminals when preflight warnings are encountered.
```

Reply:
That is a very good point !. IHMO we should not fail by default and I agree that we should use confirmY here, so that if it is not interactive then it will continue to work as usual:

```
if ! prompts_can_prompt ; then
        echo "N"
        logWarn "Automatically declining prompt, shell is not interactive"
        return 1
    fi
```

Reply:
I pushed the change.